### ISSUE TITLE
#footer slot not working on Vue 3 (next version)

### ISSUE THREAD

**Issue by antk0m** [nathanreyes/v-calendar#889]
Hello!
It seems to be easy to set `#footer` slot. I saw some examples here, but i have some problems with it...
My code:
```
  <DatePicker
    v-model="modelValue"
    :is-range="range"
    class="datepicker"
    :masks="{ weekdays: 'WWW' }"
    @update:modelValue="$emit('update:modelValue', modelValue)">
    <!-- trim-weeks not working now -->
    <template #default="{ inputValue, togglePopover }">
      <BaseInput
        v-if="range"
        @click="togglePopover"
        innerIcon="icon_clock"
        :modelValue="inputValue.start && inputValue.end ? inputValue.start + ' - ' + inputValue.end : ''"
        :placeholder="placeholder"
        :disabled="disabled"
        :readonly="true"
        :label="label"
      />
    </template>
    <template #footer>
      <div>
        <p>Please work!</p>
      </div>
    </template>
  </DatePicker>
```

Without `#footer` slot everything is ok, but i need it, and if i add the slot, it renders (in html! to the `.vc-pane-container`) some kind of error message instead of expected content. 
Error text:
```
(...args) =&gt; {
        // If a user calls a compiled slot inside a template expression (#1745), it
        // can mess up block tracking, so by default we need to push a null block to
        // avoid that. This isn't necessary if rendering a compiled `&lt;slot&gt;`.
        if (!isRenderingCompiledSlot) {
            openBlock(true /* null block that disables tracking */);
        }
        const owner = currentRenderingInstance;
        setCurrentRenderingInstance(ctx);
        const res = fn(...args);
        setCurrentRenderingInstance(owner);
        if (!isRenderingCompiledSlot) {
            closeBlock();
        }
        return res;
    }
```

I don't understand what is wrong :( Please help!

**Comment by anastasia-son** [comment]
@nathanreyes  thank you so much for such an amazing Calendar, it is the best I could find so far. 

Still having the same issue mentioned by @antk0m. I cannot set a footer slot in a Popover Date & Time Picker.
Just need it below time selects to include OK, Cancel and Set Today buttons, so that only after OK button is clicked the date would be sent to server.

**Comment by nathanreyes** [comment]
This should be patched in `3.0.0-alpha.5`
